extensions/wikia/SiteWideMessages/SpecialSiteWideMessages.php

```diff
@@ -133,30 +133,32 @@ function SiteWideMessagesSiteNoticeAfter( &$siteNotice ) {
  *
  * @author macbre
  */
-function SiteWideMessagesAddNotifications(&$skim, &$tpl) {
-	global $wgOut, $wgUser, $wgExtensionsPath;
+function SiteWideMessagesAddNotifications( Skin $skin, &$tpl ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "$tpl". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1172)